W-14758017 AEH Jan 2024 release jh (#20)
mulesoft/docs-api-experience-hub: modules/ROOT/pages/my-applications.adoc

include::_attributes.adoc[]
endif::[]

> The My Applications page lists applications that hold a contract with an API. Each client application has a client ID, client secret, description, URL, redirect URIs, and grant types.

Reviewer: What is the user task here?

Reviewer: Should My Applications be in bold?

include::_attributes.adoc[]
endif::[]

> The My Applications page enables you to review the lists applications that hold a contract with an API. Each client application has a client ID, client secret, description, URL, redirect URIs, and grant types.

Reviewer: ```suggestion
The *My Applications* page enables you to review the lists of applications that hold a contract with an API. Each client application has a client ID, client secret, description, URL, redirect URIs, and grant types.
```

APIs that are protected by a client ID enforcement policy require client applications to provide a client ID and optional client secret. The client ID and client secret credentials are automatically created when the client application is registered. For more information about registering an application, see xref:requesting-access-to-an-api-instance.adoc[Requesting Access to an API Instance].

//placeholder for screen capture of My Applications page

== Search for Applications

> If there are several applications listed in the My Applications page, search for a specific one by using the Search box. Enter the application name or description of the application and press Enter or Return. The number of applications that match your search criteria display in the list of search results.

Reviewer: ```suggestion
If the *My Applications* page lists several applications, search for a specific one by using the Search box. Enter the application name or description of the application and press Enter or Return. The number of applications that match your search criteria displays in the list of search results.
```

Lists the application’s grant types. 
* API Contracts
+

> From the API contracts section, you can see the application’s API name, environment and instances, instance URLs, SLA tier, and request access state. The environment can be either a sandbox or development environment. Request access states can be `pending, `approved, or `failed.

Reviewer: Is this necessary - describing the UI? I think the bullet list below is probably more useful because it covers the tasks that the user wants to do.

. From the *Application Details* page, edit the corresponding fields.
. Click *Save*.

=== View and Copy the Instance URL

> You can view the instance URL for an API by clicking *View URL* and copy it by clicking the *Copy from clipboard* icon.

Reviewer: ```suggestion
To view the instance URL for an API, click *View URL* and copy it by clicking *Copy from clipboard*.
```
